src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -407,6 +409,28 @@ public function fileExists(string $path, $trackContents = true): bool
         return $exists;
     }
 
+    final public function willBeAvailable(string $package, string $class, array $parentPackages): bool
+    {
+        if (!class_exists($class) && !interface_exists($class, false) && !trait_exists($class, false)) {
+            return false;
+        }
+
+        if (class_exists(FullStack::class) || !class_exists(InstalledVersions::class) || !InstalledVersions::isInstalled($package) || InstalledVersions::isInstalled($package, true)) {
```

Comment:
this looks weird to me to return `true` in case of `!InstalledVersions::isInstalled($package)`

Reply:
I think that's desired to account for weird situations when the class is here but not the package.
There is no reason why this can happen. Yet, if it happens, we should consider that the class is available IMHO.